detection_rules/utils.py

```diff
@@ -234,6 +234,13 @@ def combine_sources(*sources):  # type: (list[list]) -> list
     return event_sort(combined)
 
 
+def convert_time_span(span: str) -> int:
+    """Convert time span in datemath to value in milliseconds."""
```

Comment:
```suggestion
    """Convert time span in date math to value in milliseconds."""
```

Reply:
[Date Math](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#date-math) is proper noun. Fixing :+1:

Reply:
Fixed